ui/src/shared/components/SimpleOverlay.tsx

```diff
@@ -0,0 +1,12 @@
+import React, {SFC} from 'react'
+
+const SimpleOverlay: SFC = ({children}) => {
```

Comment:
i fear the chronogiraffe will punish us should we not name this file and function by its full honorific namesake, `SimpleOverlayTechnology`. please to appease the chronogiraffe 🙏 ⚡️

Reply:
:)